docs/meshcloud.project.md

```diff
@@ -34,6 +34,8 @@ When adding a [meshPlatform](meshcloud.platform-location.md) to a meshProject, m
 
 You can configure one or multiple [payment methods](meshcloud.project-metering.md#payment-methods) and [billing addresses](meshcloud.project-metering.md#company-billing-addresses) for you meshCustomer. For a meshProject you have to select a specific one, so the project resources can be billed via a specific payment method and to a specific billing address.
 
+You also have the option of setting an alternate payment method on a project. This feature is enabled via a configuration in meshStack. A job that runs daily will check if the already assigned payment method is expired and if it is, will replace it with the alternate payment method.
+
```

Comment:
d: what happens after payment method rotation? Is the substitute payment method moved into theh "active payment" method slot and the "new" substitute paymenet method is then empty? Please add more details